easybuild/easyconfigs/g/GCCcore/GCCcore-6.1.0_mpfr-3.1.4-allpatches-20160804.patch

```diff
@@ -0,0 +1,455 @@
+# All mpfr patches as of 2016-08-04, from same pull request for GCC 6.2.0
```

Comment:
@AFCJamie why are you duplicating this patch? There's no need to since this only patches MPFR itself, nothign in GCC, just use the existing patch for mpfr-3.1.4?